dis_snek/api/http/http_requests/members.py

```diff
@@ -102,6 +102,27 @@ async def modify_guild_member(
             reason=reason,
         )
 
+    async def modify_current_member(
+        self,
+        guild_id: "Snowflake_Type",
+        nickname: Absent[str] = MISSING,
+        reason: Absent[str] = MISSING,
+    ) -> None:
+        """
+        Modify attributes of the user
+
+        parameters:
+            nickname: The new nickname to apply
+            reason: An optional reason for the audit log
+        """
+        await self.request(
+            Route("PATCH", f"/guilds/{guild_id}/members/@me"),
+            data={
+                "nick": nickname or None,
+            },
+            reason=reason,
+        )
```

Comment:
Imo it would be better for the standard modify member method to intelligently use `@me` or `some_id` based on the id or user obejct passed